corehq/apps/cloudcare/templates/formplayer/dependencies.html

```diff
@@ -3,6 +3,11 @@
 
   <script src='https://api.mapbox.com/mapbox.js/v3.3.1/mapbox.js' async></script>
   <link href='https://api.mapbox.com/mapbox.js/v3.3.1/mapbox.css' rel='stylesheet' />
+  <script
+      src="https://browser.sentry-cdn.com/7.28.0/bundle.min.js"
+      integrity="sha384-qWZMHJ8aEyMyY977PcvgXTjW7wksFd1lNDZ1bMwpyO7Sy2Eiw3xMGCua7IfT4seD"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
I'm not a fan of using the CDN bundles but I couldn't figure out how to use the NPM versions with our JS setup since they don't include a full bundle.